policy/policy.sample.json

```diff
@@ -0,0 +1,78 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1",
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants, auto generated for POST.
+            "AppliedTo": [ # can attach multiple tenants to which the policy can be applied to.
```

Comment:
Reiterating my comment further down:

"AppliedTo" is a good name, since it says very clearly what's going to happen: The rule is going to be implemented on the hosts where those endpoints are located.

"Source" and "Destination" do not express this.